htlcswitch/interfaces.go

```diff
@@ -138,6 +138,20 @@ type ChannelUpdateHandler interface {
 	// clean. This can be used with dynamic commitment negotiation or coop
 	// close negotiation which require a clean channel state.
 	ShutdownIfChannelClean() error
+
+	// Flush is a method that disables htlc adds to the channel until it has
```

Comment:
If we have `Flush`, then do we still need the `ShutdownIfChannelClean` above?

Reply:
I'm leaning towards no. It's somewhat of an "Unsafe" method and doesn't seem necessary. It's only purpose is to interrupt the htlcManager goroutine.